frontend/src/constants/routes.ts

```diff
@@ -43,3 +43,4 @@ export const ACTIVE_ADMINFORM_RESULTS_ROUTE_REGEX = new RegExp(
   `${ADMINFORM_ROUTE}/([a-fA-F0-9]{24})/${ADMINFORM_RESULTS_SUBROUTE}(/${RESULTS_FEEDBACK_SUBROUTE})?/?`,
   'i',
 )
+export const PAYMENT_PAGE_SUBROUTE = 'payment/:paymentPageId'
```

Comment:
I was thinking that this particular Id should be agnostic/decoupled on FE. I.e., for stripe payments, it could be `pi_xyz`, and for other possible integrations like paynow, it could be `pn_xyz`.

BE should be sensitive to what this value is, and should be consistent in referencing it. I.e., if we need to change the `paymentPageId` from stripe's payment intent, to `UUID` or Mongo's `objectId` under the hood, then we should only change what BE passes to FE, and how BE reference this field. FE will treat this as a reference variable, and doesn't have to know how to manipulate or what it actually is.

Wdyt?

Reply:
> we should only change what BE passes to FE, and how BE understands this field. FE will treat this as a reference variable, and doesn't have to know how to manipulate or what it actually is.

I agree with the principle, but I'm not sure I see what advantage using the payment intent id over the payment id gives us in the fe. as an example, in the implementation we pass the payment's payment intent id to the fe from the encrypt controller and receive the payment intent id in e.g. `getPaymentInfo`. But the fe wouldn't see anything different from if we had passed the mongo object id instead. Is there somewhere in the code that, if we were to only pass the objectid, it would require some casing behavior in the fe?

Reply:
I just realized that I was arguing about a different thing 🙈 . Right, it would make the flow simpler if BE uses `paymentId` over `paymentIntentId` until the point where FE needs to communicate with `stripe`.

Reply:
Switched BE reference to `paymentId` in a9b31057